completion/available/wpscan.completion.bash

```diff
@@ -1,15 +1,62 @@
 #!/usr/bin/bash
 
-if command -v wpscan > /dev/null; then
-    __wpscan_completion()  {
-        local OPTS=("--help --hh --version --url --ignore-main-redirect --verbose --output --format --detection-mode --scope --headers --user-agent --vhost --random-user-agent --user-agents-list --http-auth --max-threads --throttle --request-timeout --connect-timeout --disable-tlc-checks --proxy --proxy-auth --cookie-string --cookie-jar --cache-ttl --clear-cache --server --cache-dir --update --no-update --wp-content-dir --wp-plugins-dir --wp-version-detection --main-theme-detection --enumerate --exclude-content-based --plugins-list --plugins-detection --plugins-version-all --plugins-version-detection --themes-list --themes-detection --themes-version-all --themes-version-detection --timthumbs-list --timthumbs-detection --config-backups-list --config-backups-detection --db-exports-list --db-exports-detection --medias-detection --users-list --users-detection --passwords --usernames --multicall-max-passwords --password-attack --stealthy")
-        COMPREPLY=()
-        for _opt_ in ${OPTS[@]}; do
-            if [[ "$_opt_" == "$2"* ]]; then
-                COMPREPLY+=("$_opt_")
-            fi
-        done
+if _command_exists wpscan > /dev/null; then
+
+    _wpscan_completions()   {
```

Comment:
Is there a reason this name was changed? If not, could you preserve the original function name `__wpscan_completion`? It's the convention most bash completions use and I'd like to stick with it.

@NoahGorny Maybe this is something we should add to the docs?

Reply:
From `contributing.rst`:
```rst
* Internal functions that aren't to be used by the end user should start with an underscore, e.g. ``_my-new-internal-function``.
```

Maybe we should improve this section

Reply:
@cornfeedhobo Done